### PR TITLE
Update with erad information

### PR DESCRIPTION
**Pull request by mgrover1**
Addresses #1 - use ERAD relevant logos and links on the page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,18 +9,42 @@ This tutorial covers how to get started with the Open Radar Science stack!
 
 ## Motivation
 
-The course will discuss the principles of open science and provide an overview of the most mature and exciting open-source software packages available for radar data processing. Additionally it will provide guidelines to collaborate in or start a new open project. It will have a hands-on approach and the students will be able to use the software packages themselves. Anyone with interest in using or developing open source software tools is welcomed.
+The course will take place on August 28th 2012, right before the [2022 ERAD Radar Conference]([hhttps://cscenter.co.jp/icrm2019/](https://www.erad2022.ch/)). We will introduce the participants to [community software packages](http://openradarscience.org) designed for radar data processing, including (but not limited to) [BALTRAD](http://git.baltrad.eu/), [Py-ART](http://arm-doe.github.io/pyart/), and [wradlib](https://wradlib.org). Following a welcome, there will be an introduction to Open Science concepts and the Open Radar context.
```
[the remark this answers is not in the document]
Comment (mgrover1) [reply README.md]: Yup! That link along with some of the readme links were bad... which was causing the CI to file (the link checker)

**Review by kmuehlbauer — APPROVED** [review]
@mgrover1 Merge at your discretion. We can easily adapt further with follow-up PR's.

File: README.md [hunk 2]
```diff
@@ -32,14 +32,14 @@ The common ground for most of those tools is Python, so we'll feature a quick in
...
 ## Course program
 * 09:00 - 09:15 Welcome and getting started
-* 09:15 - 09:45 [Community weather radar software and Open Science](overview-openscience)
+* 09:15 - 09:45 Community weather radar software and Open Science
 * 09:45 - 10:30 Overview of the open source radar processing packages
 * 10:30 - 11:00 Coffee break
-* 11:00 - 11:45 [Hands on Py-ART](pyart)
-* 11:45 - 12:30 [Hands on wradlib](wradlib)
+* 11:00 - 11:45 Hands on Py-ART
+* 11:45 - 12:30 Hands on wradlib
```
[the remark this answers is not in the document]
Comment (mgrover1) [reply README.md:39]: yeah - should I add in those directories now? Or add in with a different PR?

[the remark this answers is not in the document]
Comment (mgrover1) [reply README.md:39]: We can submit additional PRs!

**Pull request merged**